src/dswx_sar/save_mgrs_tiles.py

```diff
@@ -1,73 +1,114 @@
 import os
 import time
-import numpy as np
 import mgrs
 import logging
 import mimetypes
-from osgeo import gdal, osr
-from pyproj import CRS
 import h5py
 import glob
-import mgrs
+import ast
```

Comment:
Please make sure that `ast`, `geopandas`, and these other libraries are added as a dependency of the SAS

Reply:
Yes. They are added.